code/espurna/garland/animations/anim_glow.h

```diff
@@ -22,7 +22,7 @@ class AnimGlow : public Anim {
                 glowForEachLed(i);
             }
         } else {
-            for (int i = 0; i < numLeds; ++i) {
+            for (int i = numLeds - 1 ; i >= 0; --i) {
```

Comment:
I wonder if begin() & rbegin() alternatives would clearer? i.e.
```cpp
auto begin = leds.begin();
auto end = leds.end();
while (begin != end) {
... // something that accepts Color&
}
```

There's also a way to bind method at runtime a-la virtual function:
https://godbolt.org/z/n3jaKrWYT

Reply:
glowForEachLed(i) requires using counter, so using iterator can be more complex in this case.

Reply:
True... but given it is a vector, it is a simple pointer arithmetic via `it - leds.begin()` or `leds.end() - it`, since size of the object is known beforehand
Or `std::distance`. Either works though